examples/EnableServiceProfilerForAzureFunction/appsettings.json

```diff
@@ -0,0 +1,13 @@
+{
```

Comment:
Could you please double check to make sure the appsettings.json is used? It could be by default but I think it a good idea to verify it.

Reply:
Updated the program file to use appsettings instead of secrets since it seems easier to set the values that way